game-app/game-core/src/main/java/games/strategy/engine/data/Unit.java

```diff
@@ -62,7 +62,7 @@ public class Unit extends GameDataComponent implements DynamicallyModifiable {
   // the transport that is currently transporting us
   private Unit transportedBy = null;
   // the units we have unloaded this turn
-  private List<Unit> unloaded = List.of();
```

Comment:
I don't think these can change without breaking save games.

Reply:
@asvitkine Sorry, that wasn't so smart. I have've reverted it again.